packages/cadl-python/test/generated/hello/azure/hello/_configuration.py

```diff
@@ -24,7 +24,7 @@ class HelloClientConfiguration(Configuration):  # pylint: disable=too-many-insta
     def __init__(self, **kwargs: Any) -> None:
         super(HelloClientConfiguration, self).__init__(**kwargs)
 
-        kwargs.setdefault("sdk_moniker", "helloclient/{}".format(VERSION))
+        kwargs.setdefault("sdk_moniker", "hello/{}".format(VERSION))
```

Comment:
why did the name change here

Reply:
it changes because of current logic: https://github.com/Azure/autorest.python/blob/ea22c0f39edafa3df2bb80680cfc09f36d217369/packages/autorest.python/autorest/codegen/models/client.py#L354-L358